pkg/ingester/stream_test.go

```diff
@@ -312,6 +313,9 @@ func TestPushRateLimit(t *testing.T) {
 	l := validation.Limits{
 		PerStreamRateLimit:      10,
 		PerStreamRateLimitBurst: 10,
+		ShardStreams: &shardstreams.Config{
```

Comment:
nit: Because whole stream rate limiting is controlled by the flag we pass to `Push`, there's no need to set this up here, either.